### PR TITLE
Chat message

### PR DESCRIPTION
**Pull request by emir4ik-code**

### CODE REVIEW

**Comment by emir4ik-code** [comment]
chatMessageReceiver

**Comment by emir4ik-code** [comment]
chatMessageSender

**Comment by emir4ik-code** [comment]
Option icons appear on hover

**Review by ntan03 — APPROVED** [review]

File: src/features/chat/components/ChatMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import ReplyIcon from "@mui/icons-material/Reply";
+import { Box, Typography, Avatar } from "@mui/material";
+import useTheme from "features/theme/useTheme";
+import { useState } from "react";
+
+type ChatMessageProps = {
+  isAuthUser: boolean;
+  content: string;
+  avatar: string;
```
Comment (raylu511) [inline src/features/chat/components/ChatMessage.tsx:11]: I would update this to profilePic so we don't have to transform the property name we get from the response

File: src/features/chat/components/ChatMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              margin: 0,
+              padding: 0,
+              boxSizing: "border-box",
+            }}
+          >
+            <Typography sx={{ fontSize: "16px", color, padding: "10px" }}>
+              {content}
+            </Typography>
+          </Box>
+          <Box
```
Comment (raylu511) [inline src/features/chat/components/ChatMessage.tsx:87]: this could be self closing

**Review by raylu511 — APPROVED** [review]

**Review by ntan03 — APPROVED** [review]

**Pull request merged**